fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -3,11 +3,11 @@ import Ember from 'ember';
 export default Ember.Route.extend({
   model() {
       var deployment = this.modelFor('deployment');
-      var deployTaskPromise = this.store.query('foreman-task', {search: "id = " + deployment.get('foreman_task_uuid')});
```

Comment:
this was a work-around with ember-data 0.19-beta where it was only returning what was in the store and not in the db. With Ember-Data 1.13.x, `this.findRecord` should return from the DB.